### PR TITLE
Frob everything that could make it into a splice or tufo valu.

### PR DESCRIPTION
**Pull request by aheinz-fe**

### CODE REVIEW

File: synapse/cores/common.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def _onSetCaching(self, valu):
 
     def _reqSpliceInfo(self, act, info, prop):
         valu = info.get(prop)
-        if prop == None:
+        if valu == None:
```
Comment (aheinz-fe) [inline synapse/cores/common.py]: Bonus bug fix?

Comment (williballenthin) [inline synapse/cores/common.py]: should make comparisons against `None` like:

    if valu is None:

Comment (aheinz-fe) [inline synapse/cores/common.py]: I made this same argument when I first started working on synapse, but @invisig0th had some prior experience with some sort of python bug where the None reference wasn't always the same?

**Comment by invisig0th** [comment]
closing this one for the splice rematch as well :)  this idea will definitely be incorporated :)

**Pull request closed without merging**